src/resources/Workers.php

```diff
@@ -25,7 +25,8 @@ public function __construct($api)
 			'insertTask' =>  ['method' => 'PUT', 'path' => '/containers/workers/:workerId'],
 			'getSchedule' =>  ['method' => 'GET', 'path' => '/workers/:workerId/schedule'],
 			'setSchedule' => ['method' => 'POST', 'path' => '/workers/:workerId/schedule'],
-			'matchMetadata' => ['method' => 'POST', 'path' => '/workers/metadata']
+			'matchMetadata' => ['method' => 'POST', 'path' => '/workers/metadata'],
+			'listAssignedTasks' => ['method' => 'GET', 'path' => '/workers/:workerId/tasks', 'queryParams' => true]
```

Comment:
We had named this method `getTasks` in the python wrapper so, we should keep the same names across the wrappers, shouldn't we?